libs/api/domains/financial-statements-inao/src/lib/financialStatementsInao.resolver.ts

```diff
@@ -4,10 +4,16 @@ import { Query, Resolver } from '@nestjs/graphql'
 import { IdsUserGuard, ScopesGuard } from '@island.is/auth-nest-tools'
 
 import { Election } from './models/election.model'
+import { FinancialStatementsInaoService } from './financialStatementsInao.service'
+import { ClientType } from './models/clientType.model'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
 @Resolver()
 export class FinancialStatementsInaoResolver {
+  constructor(
```

Comment:
Shouldn't all operations in this class at least be protected by the internal scope?

Reply:
Added internal api scope to class.